docker/Dockerfile.xpu

```diff
@@ -14,6 +14,7 @@ RUN apt clean && apt-get update -y && \
     libxext6 \
     libgl1 \
     lsb-release \
+    libaio-dev \
```

Comment:
what's this for? if nixl have libaio dependency, I am not sure whether this is a good manner since version may matter.